File: src/kotlin/ast/KoKoId.java
/**
 @author loriacarlos@gmail.com
 
*/
package kokoslan.ast;


public class KoKoId : KoKoAtom<String> ( value: String ): super(value){
	
   override 
   fun eval(KoKoContext ctx): KoKoValue{
	   return ctx.find(this);
   }
   
}

File: src/kotlin/ast/KoKoId.kt
/*
  @author Denis Rodriguez Viquez
          Luis Vasquez Quiros
          Walter Chavez Oviedo
  @since 2017
*/
package kokoslan.kotlin.ast

import kokoslan.kotlin.eval.*

class KoKoId( value : String) : KoKoAtom<String>( value ){
	override fun eval(ctx : KoKoContext) = ctx.find(this)
}